standard/docs/en/extensions/index.md

```diff
@@ -14,66 +14,23 @@ All core and community extensions are listed in the [extensions registry](https:
 
 ## Using extensions
 
-Extensions are applied by adding their URLs to the `extensions` array in the release or record package. To use one or more extensions, select them from the extension lists below and include the following in your release or record package metadata:
-
-<style><!--
-#using-extensions div[class^='highlight'] pre {
-   white-space: pre-wrap;  
-   white-space: -moz-pre-wrap;  /* Mozilla, since 1999 */
-   white-space: -pre-wrap;      /* Opera 4-6 */
-   white-space: -o-pre-wrap;    /* Opera 7 */
-   word-wrap: break-word;       /* Internet Explorer 5.5+ */
-}
---></style> 
-
-```json
-{
-    "extensions":[],
-    "releases":[]
-}
-```
+Extensions are applied by adding their URLs to the `extensions` array in the release package or record package. You can discover extensions and read their documentation using the [Extension Explorer](https://extensions.open-contracting.org/en/).
 
-## Core extensions
+This version of OCDS uses these specific versions of core extensions:
```

Comment:
This page still uses the terminology of 'core' extensions but the extensions explorer uses 'recommended'

Reply:
Changing that language in the OCDS docs would be something for a later release.